docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -291,3 +291,19 @@ considerations apply:
 
 Additional considerations about errors that apply only to a specific Runtime Hook will be documented in the hook-specific 
 implementation documentation.
+
+## Tips & tricks
+
+After you implemented and deployed a Runtime Extension you can manually test it by sending HTTP requests.
+This can be for example done via kubectl:
+
+```bash
+# Send a Discovery Request to the webhook-service in namespace default with protocol https on port 443:
+kubectl create --raw /api/v1/namespaces/default/services/https:webhook-service:443/proxy/hooks.runtime.cluster.x-k8s.io/v1alpha1/discovery --v=8 -f <(echo '{"kind":"DiscoveryRequest","apiVersion":"hooks.runtime.cluster.x-k8s.io/v1alpha1"}')
+
+# Open a proxy with kubectl and then play around with curl:
+## First terminal:
+kubectl proxy
+## Second terminal:
+curl -X [GET/POST/...] 127.0.0.1:8001/api/v1/namespaces/default/services/https:webhook-service:443/proxy/hooks.runtime.cluster.x-k8s.io/v1alpha1/discovery
```

Comment:
Maybe we could outline what requests are actually made Runtime SDK (I assume POST including some json input!?

Reply:
Good point, done